scripts/emitters.py

```diff
@@ -0,0 +1,117 @@
+from itertools import product
```

Comment:
Include a header about the content and purpose of the script

Reply:
Done. :+1: